bin/install

```diff
@@ -453,7 +453,6 @@ end
 # Verify we have a sane environment that OMF can run in.
 function sane_environment_check
   say "Checking for a sane environment..."
-  assert_cmds
```

Comment:
we should keep `assert_cmds` here otherwise we wont' be checking other commands. since git isn't aborting anymore this should be fine.